trajectorytools/trajectories/concatenate.py

```diff
@@ -64,14 +66,54 @@ def _concatenate_idtrackerai_dicts(traj_dicts):
     return traj_dict_cat
 
 
-def from_several_idtracker_files(trajectories_paths, **kwargs):
+def pick_trajectory_file(session_folder):
+    """Select the best trajectories file
+    available in an idtrackerai session
+    """
+    trajectories = os.path.join(session_folder, "trajectories", "trajectories.npy")
+    trajectories_wo_gaps = os.path.join(session_folder, "trajectories_wo_gaps", "trajectories_wo_gaps.npy")
+
+    if os.path.exists(trajectories_wo_gaps):
+        return trajectories_wo_gaps
+    elif os.path.exists(trajectories):
+        return trajectories
+    else:
+        raise Exception(f"Session {session_folder} has no trajectories")
+
+
+def is_idtrackerai_session(path):
+    """Check whether the passed path is an idtrackerai session
+    """
+    return re.search("session_.*", path) and os.path.isdir(path)
+
+def get_trajectories(idtrackerai_collection_folder):
+    """Return a list of all trajectory files available in an idtrackerai collection folder
```

Comment:
Maybe be more specific about what an idtrackerai collection folder is:

Is this a folder that contains subfolders with videos and other session_ subfolders?

Reply:
Yes, that is how I am organizing my data. In other words, it is an `imgstore.VideoImgStore` where several `session_` folders are also present.